### ISSUE TITLE
Occasional NullPointerException on world creation

### ISSUE THREAD

**Issue by eviltak** [Terasology/Minesweeper#4]
A `NullPointerException` in the `Chunk.setBlock` method call (line 57) in the `generateChunk` method of the `MineRasterizer` occasionally manifests itself on world creation. On further inspection, it turns out that the passed `block` argument to the `setBlock` method is `null`, which in turn is seemingly a direct consequence of #3. Should be fixed with resolution of #3.

## Relevant logs
```
23:24:28.850 [Chunk-Generator-6] ERROR o.t.u.concurrency.TaskProcessor - Error in thread Chunk-Generator-6
java.lang.NullPointerException: null
	at org.terasology.world.chunks.internal.ChunkImpl.setBlock(ChunkImpl.java:162)
	at org.terasology.world.chunks.internal.ChunkImpl.setBlock(ChunkImpl.java:173)
	at org.terasology.MineSweeper.generator.MineRasterizer.generateChunk(MineRasterizer.java:64)
	at org.terasology.world.generation.WorldImpl.rasterizeChunk(WorldImpl.java:64)
	at org.terasology.world.generation.BaseFacetedWorldGenerator.createChunk(BaseFacetedWorldGenerator.java:66)
	at org.terasology.world.chunks.localChunkProvider.LocalChunkProvider$1.run(LocalChunkProvider.java:620)
	at org.terasology.utilities.concurrency.TaskProcessor.run(TaskProcessor.java:50)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
23:24:28.853 [Chunk-Generator-4] ERROR o.t.u.concurrency.TaskProcessor - Error in thread Chunk-Generator-4
java.lang.NullPointerException: null
	at org.terasology.world.chunks.internal.ChunkImpl.setBlock(ChunkImpl.java:162)
	at org.terasology.world.chunks.internal.ChunkImpl.setBlock(ChunkImpl.java:173)
	at org.terasology.MineSweeper.generator.MineRasterizer.generateChunk(MineRasterizer.java:57)
	at org.terasology.world.generation.WorldImpl.rasterizeChunk(WorldImpl.java:64)
	at org.terasology.world.generation.BaseFacetedWorldGenerator.createChunk(BaseFacetedWorldGenerator.java:66)
	at org.terasology.world.chunks.localChunkProvider.LocalChunkProvider$1.run(LocalChunkProvider.java:620)
	at org.terasology.utilities.concurrency.TaskProcessor.run(TaskProcessor.java:50)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
```

**Comment by pollend** [comment]
@eviltak when it's up you should be able to grab it from this link: https://codein.withgoogle.com/tasks/6320382694916096/

**Comment by eviltak** [comment]
@pollend thanks! I see that there is a different unclaimable task at the link you provided, so I'll check back later to see when the task will be up.

**Comment by pollend** [comment]
@eviltak here you go https://codein.withgoogle.com/tasks/6033853514776576/

**Comment by eviltak** [comment]
Closed via #5.
